app/views/equipment_items/_equipment_item.html.erb

```diff
@@ -0,0 +1,17 @@
+<% #TODO check if this can be removed %>
```

Comment:
I'm going to double check that we don't render this partial anywhere and then we can just delete this file. I'll keep you posted.